tests/setup/profile_minimal/doc_test.go

```diff
@@ -43,5 +43,13 @@ values:
   pilot:
     env:
       PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING: true
+components:
+  egressGateways:
+  - enabled: false
+    name: istio-egressgateway
+  ingressGateways:
+  - enabled: false
+    name: istio-ingressgateway
 `
+	cfg.DeployEastWestGW = false
```

Comment:
Also not needed since this one should be false by default

Reply:
Why did you add these? Am I missing something?

Reply:
Hmm, maybe there's a bug here: https://github.com/istio/istio/blob/7717d61314a9880fbd0c7742f1622f3070241944/pkg/test/framework/components/istio/kube.go#L422

I think it should also be checking for multicluster:
```
		if !i.cfg.DeployEastWestGW || !i.ctx.Environment().IsMultiCluster() {
```

Reply:
Yes Frank there is an issue with how the framework starts it seems. @kfaseela was going to look into it some more. The framework sets up the cluster using serveral iop files. Two are from the framework and the third is created by what we setup in our files like specific profiles. 

The first few files set the ingress and egressgteway values to true so even though the profile is minimal, the gateways are started. We need to specifically set them to false to prevent them from starting.

I'm not even convinced that we are starting things correctly in the other tests.

Reply:
Here's what's being used:
```
	Installing Istio components on cluster cluster-0: InFilenames:      [/work/tests/integration/base.yaml /work/tests/integration/iop-integration-test-defaults.yaml /work/out/darwin_amd64/tests-setup-profile-minimal-d18/_suite_context/istio-deployment-3202578840/iop.yaml]
```
Even though the last yaml has the minimal profile specified, the gateways are installed. This might be an issue with Istio that needs to be fixed.

Reply:
Note that the tests/integration files come from istio/istio.

Reply:
Although the EastWest might not need to be called out specifically to disable.

Reply:
@frankbu better to fix the framework than doing it this way?

Reply:
I would recommend fixing the framework. The suggested change for DeployEastWestGateway that I mentioned above seems like an obvious omission. I would also think that if the gateways are being installed with profile=minimal (that's probably just another omission in the framework's config file), then we should definitely fix that. Otherwise the framework is not testing what the user intends. Hacking in fixes like this is not a good approach.

Reply:
Makes sense @frankbu ! @ericvn I will try to fix the framework first, and then get back to this PR

Reply:
It seems that the EastWestGW is also used for VM service testing, not just multicluster. Since the VM integration tests are not cleanly separated like the multicluster ones are, it doesn't look like we can easily change the default test cluster config to not include EastWestGW.

So, I think the best thing to do here is configure the profile_minimal explicitly, like you have done. But, remove `profile=minimal` (line 41) from the configuration, since it's confusing and doesn't work as expected anyway.

Reply:
@frankbu done